### PR TITLE
KDE: a very short report for 2024q2

### PR DESCRIPTION
**Pull request by adriaandegroot**

### CODE REVIEW

File: website/content/en/status/report-2024-04-2024-06/kde.adoc
Comment (BSDKaffee) [inline website/content/en/status/report-2024-04-2024-06/kde.adoc:1]: Qt6/PySide6 are now at 6.7.2. :)

Comment (BSDKaffee) [inline website/content/en/status/report-2024-04-2024-06/kde.adoc:1]: We're also at CMake 3.29.6 in this quarter.

Comment (adriaandegroot) [reply website/content/en/status/report-2024-04-2024-06/kde.adoc:1]: All true.

File: website/content/en/status/report-2024-04-2024-06/kde.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The KDE on FreeBSD project packages CMake, Qt, and software from the KDE Community, for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma (for both X11 and Wayland) and hundreds of applications that can be used on any FreeBSD machine.
+The mailto:kde@FreeBSD.org[KDE team] is part of mailto:desktop@FreeBSD.org[desktop@], building the software stack to make FreeBSD beautiful and usable as a daily-driver graphical desktop workstation.
+
+We missed a quarter, while the broader KDE community celebrated the KDE Megarelease of KDE Frameworks 6, KDE Plasma 6 and KDE Gear.
+The latest software is still not available on FreeBSD, pending more testing and some integration work.
+
+
+
```
Comment (pauamma) [inline website/content/en/status/report-2024-04-2024-06/kde.adoc:17]: Spurious line breaks.

**Review by pauamma — APPROVED** [review]
Approved with the 2 spurious line breaks removed.

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**